src/javaRosa.js

```diff
@@ -1760,13 +1768,28 @@ define([
                 },
                 lstring: "Help Message",
                 widget: function (mug, options) {
-                    return itextLabelBlock(mug, $.extend(options, {
-                        itextType: "help",
-                        getItextByMug: function (mug) {
-                            return mug.p.helpItext;
-                        },
-                        displayName: "Help Message"
-                    }));
+                    var block = itextLabelBlock(mug, $.extend(options, {
+                            itextType: "help",
+                            getItextByMug: function (mug) {
+                                return mug.p.helpItext;
+                            },
+                            displayName: "Help Message"
+                        })).on('change', function() {
+                            if (!HELP_MARKDOWN) {
+                                return;
+                            }
+                            var mug = this.mug,
+                                helpItext = mug.p.helpItext,
+                                helpItextForm = helpItext.forms[0],
+                                markdownForms = _.find(helpItext.forms, function(itext) {
+                                    return itext.name === 'markdown';
+                                });
+                            if (markdownForms) {
+                                markdownForms.data = JSON.parse(JSON.stringify(helpItextForm.data));
```

Comment:
Would `_.clone(helpItextForm.data)` be appropriate here? That's how it's done in [`ItextForm.clone()`](https://github.com/dimagi/Vellum/blob/markdown/src/javaRosa.js#L176)